nevergrad/benchmark/experiments.py

```diff
@@ -1138,7 +1140,6 @@ def rocket(seed: tp.Optional[int] = None, seq: bool = False) -> tp.Iterator[Expe
                             yield xp
 
 
-@registry.register
```

Comment:
why is this removed? Can we remove the xp?

Reply:
I unregistered so that it is no more in the dashboard because mujoco = reproducibility issues, but we might keep it for now as there are users who like it. Yeah I guess unregistering was wrong.

Reply:
users can use it if it's unregistered :s so it's better to keep it registered or delete altogether

Reply:
Thanks for having brought them back.